README.md

```diff
@@ -22,6 +22,12 @@ An implementation of VHD [VHD specification](https://technet.microsoft.com/en-us
     2. https://github.com/Azure/azure-sdk-for-go
             This package is used to make Microsoft Azure storage API calls.
 
+# Features
+
+    1. Fast uploads - This tool offers faster uploads than the [Azure Xplat CLI](https://github.com/Azure/azure-xplat-cli).
```

Comment:
Yeah... I thought Markdown was smart enough to not code-block a numbered list, but you're definitely right. I'll unindent. I also used "Azure Cross Platform Command Line" as the branding -- that's what they have on their GitHub repo.